enterprise_gateway/services/processproxies/processproxy.py

```diff
@@ -47,6 +47,13 @@
 
 local_ip = localinterfaces.public_ips()[0]
 
+class KernelChannel(Enum):
+    SHELL = "SHELL"
+    IOPUB = "IOPUB"
+    STDIN = "STDIN"
+    HEARTBEAT = "HB"
+    CONTROL = "CONTROL"
+    COMMUNICATION = "EG_COMM"
```

Comment:
I think we should add a comment for the EG_COMM channel, something like...
`# Optional channel created by remote launcher used to issue remote interrupts - not a ZMQ channel`
so that its not confused as a ZMQ construct.